packages/jsii-pacmak/test/expected.jsii-calc/python/src/jsii_calc/__init__.py

```diff
@@ -1367,15 +1367,15 @@ def derived_to_first(self, *, another_required: datetime.datetime, bool: bool, n
 
         Arguments:
             derived: -
-            anotherRequired: 
+            another_required: 
             bool: 
-            nonPrimitive: An example of a non primitive property.
-            anotherOptional: This is optional.
-            optionalAny: 
-            optionalArray: 
+            non_primitive: An example of a non primitive property.
+            another_optional: This is optional.
+            optional_any: 
```

Comment:
What do we think about the params without text? how does this render? Should we add some placeholder text?

Reply:
Well, the current code returns a ``-`` character if there are not docs for the argument but that causes things to render funny, as in https://docs.aws.amazon.com/cdk/api/latest/python/aws_cdk.aws_ecs/Ec2Service.html#aws_cdk.aws_ecs.Ec2Service.add_placement_constraints.  I think that should be changed, either to a blank character, empty string, or some other default string.

Reply:
Not sure I follow... So this is okay?